src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java

```diff
@@ -121,6 +123,10 @@ private String getNodeSuffix(Project project, String affectedFilePath) {
             parts.add(String.format("%s draft%s", draftsForFile.size(), draftsForFile.size() == 1 ? "" : "s"));
         }
 
+        if(reviewed.get().contains(fileName)) {
```

Comment:
NIT: please add a space after `if`

Reply:
Hint with the suggest (ctrl+g) functionality this can be easily applied.

```suggestion
        if (reviewed.get().contains(fileName)) {
```

Reply:
This re-appeared (and I'm not able to apply this hint because I do not have write access to your fork). After this one is resolved, I'll merge.